packages/bot-web-ui/src/stores/run-panel-store.js

```diff
@@ -482,9 +537,27 @@ export default class RunPanelStore {
 
     @action.bound
     onUnmount() {
+        const { journal, transactions } = this.root_store;
+
         RunPanelStore.unregisterBotListeners();
         this.disposeIsSocketOpenedListener();
 
+        if (typeof this.disposeStatisticsListener === 'function') {
+            this.disposeStatisticsListener();
+        }
+
+        if (typeof this.disposeSwitchAccountListener === 'function') {
+            this.disposeSwitchAccountListener();
+        }
```

Comment:
Since we're creating a `disposeListeners` for both `journal` and `transactions`, why not create one for `run_panel` too?